src/Umbraco.Web/Models/Mapping/MemberModelMapper.cs

```diff
@@ -477,8 +481,41 @@ public MemberDisplay Convert(ResolutionContext context)
                 return AutoMapperExtensions.MapWithUmbracoContext<IMember, MemberDisplay>(member, context.GetUmbracoContext());
             }
         }
-    }
-}
 
+        internal class MemberSensitivePropertiesResolver : SensitivePropertiesResolver<IMember>
+        {
 
+            /// <summary>
+            /// Overridden to assign the IsSensitive property values
+            /// </summary>
+            /// <param name="umbracoContext"></param>
+            /// <param name="content"></param>
+            /// <param name="properties"></param>
+            /// <returns></returns>
+            protected override List<ContentPropertyDisplay> MapProperties(UmbracoContext umbracoContext, IContentBase content, List<Property> properties)
+            {
+                var result = base.MapProperties(umbracoContext, content, properties);
+                var member = (IMember)content;
+                var memberType = member.ContentType;
 
+                //now update the IsSensitive value
+                foreach (var prop in result)

```

Comment:
This code block is a duplicate of what is already happening in `MemberTabsAndPropertiesResolver` so we can definitely re-use that code somewhere